Pod/Classes/BONTag_Private.h

```diff
@@ -0,0 +1,49 @@
+//
+//  BONTag_Private.h
+//  Pods
+//
+//  Created by Nora Trapp on 5/11/16.
+//
+//
+
+#import "BONTag.h"
+
+@interface BONTag ()
+
+@property (strong, nonatomic, BONNonnull) BONGeneric(NSMutableArray, NSValue *) * ranges;
+
+/**
+ *  Finds all escaped tags within a given string.
+ *
+ *  @param string                The string to search.
+ *  @param tags                  The tags to search for.
+ *  @param stripEscapeCharacters If YES, the escape characters will be removed from the string and the resulting ranges.
+ *
+ *  @return An array of ranges representing the escaped tags.
+ */
++ (BONNonnull BONGeneric(NSArray, NSValue *) *)escapedRangesInString:(NSString *BONCNonnull *BONCNonnull)string withTags:(BONNonnull BONGeneric(NSArray, BONTag *) *)tags stripEscapeCharacters:(BOOL)stripEscapeCharacters;
```

Comment:
#req document the fact that `*string` is mutated to contain the version of the string with unescaped tags stripped.